packages/react-data-grid/src/Canvas.tsx

```diff
@@ -138,9 +138,6 @@ export default class Canvas extends React.PureComponent<Props> {
   }
 
   getRows(rowOverscanStartIdx: number, rowOverscanEndIdx: number) {
-    if (Array.isArray(this.props.rowGetter)) {
-      return this.props.rowGetter.slice(rowOverscanStartIdx, rowOverscanEndIdx);
-    }
```

Comment:
I did not find a case where `rowGetter` is an array so I am deleting this logic. Also in many places we assume `rowGetter` is a function